docs/.local_build.sh

```diff
@@ -12,14 +12,12 @@ mkdir -p ../_dist
 rsync -ruv . ../_dist
 cd ../_dist
 poetry run python scripts/model_feat_table.py
-poetry run nbdoc_build --srcdir docs --pause 0
-mkdir docs/templates
-cp ../templates/docs/INDEX.md docs/templates/index.md
```

Comment:
why get rid of this? doesn't build not work without it

Reply:
The templates/index.md copy? I think it's duplicated below, and also needs a `-p` if the folder already exists